quickstart/src/index.js

```diff
@@ -83,7 +83,8 @@ $.getJSON('/token', function(data) {
 
 // Successfully connected!
 function roomJoined(room) {
-  activeRoom = room;
+  window.room = room;
+  window.activeRoom = activeRoom = room;
```

Comment:
Are `window.room` and `window.activeRoom` intentional? Should we go with one or the other?